packages/frontend/src/components/attachment/mediaAttachment.tsx

```diff
@@ -609,44 +609,61 @@ export function WebxdcAttachment({
     tabIndex: rovingTabindex.tabIndex,
   } as const
 
-  if (loadResult.kind === 'loadingError') {
+  const webxdcInfoFetch = useRpcFetch(
+    BackendRemote.rpc.getWebxdcInfo,
+    loadResult.kind === 'message' ? [accountId, messageId] : null
+  )
+  if (webxdcInfoFetch?.result?.ok === false) {
+    log.error(
+      `Failed to load webxdc info for message:
+      messageId: ${messageId},
+      accountId: ${accountId},
+      error: ${webxdcInfoFetch.result.err}`
+    )
+  }
+  const webxdcInfo = webxdcInfoFetch?.result?.ok
+    ? webxdcInfoFetch.result.value
+    : null
+
+  if (webxdcInfoFetch?.loading) {
     const onContextMenu = getBrokenMediaContextMenu(
       contextMenu.openContextMenu,
       openDialog,
       deleteMessage,
       messageId,
       accountId
     )
-
     return (
       <div
         ref={interactiveElRef}
-        className={'media-attachment-webxdc broken ' + rovingTabindex.className}
-        title={loadResult.error}
+        className={'media-attachment-webxdc ' + rovingTabindex.className}
```

Comment:
> only thing I'm not really sure about is if broken media handling in Gallery is still the same as before, as I didn't had the various use cases

Indeed it's not the same.

- `getBrokenMediaContextMenu` is used for the "loading" state.
- `loadResult.error` is not used anymore.
- `tx('attachment_failed_to_load')` is not used, intead 'webxdcInfo is not available' is shown in both cases.

But it's not super important I guess.